src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -68,6 +71,24 @@ def get_file_content(self, filepath_name):
         response_json = self.request_handler.perform_request("GET", custom_args)
         return response_json
 
+    def delete_uss(self, filepath_name):
+        """
+        Delete a file or directory
+
+        Parameters
+        ----------
+        filepath of the file to be deleted
+
+        Returns
+        -------
+        204
+            HTTP Response for No Content
+        """
+        custom_args = self.create_custom_request_arguments()
+        custom_args["url"] = "{}fs{}".format(self.request_endpoint,filepath_name)
```

Comment:
The `filepath_name` should be supported either with or without a leading slash. Could you change it to something like this:
```custom_args["url"] = "{}fs/{}".format(self.request_endpoint, filepath_name.lstrip("/"))```